compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEBufferizeToAllocation.cpp

```diff
@@ -39,6 +40,23 @@ static LogicalResult applyBufferizeToAllocation(RewriterBase &rewriter,
   return success();
 }
 
+/// Utility to fetch operands from the LinalgOp's input and output.
+static SmallVector<Value> getInputOutputOperands(linalg::LinalgOp &linalgOp) {
```

Comment:
The name and comment is confusing. It seems to suggest you are getting the input and output of a linalg op, but that isnt what this is doing.

Reply:
Updated the comments and changed function name `isMatmulElementwise` -> `isMatmulProducerOfElementwise`.